README.md

```diff
@@ -405,21 +405,19 @@ await octokit.rest.issues.create({
 });
 ```
 
-The REST API endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID or parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
+The `octokit.rest` endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID and parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
 
-Every endpoint method is an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
+Under the covers, every endpoint method is actually an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
 
 #### Arbitrary requests
 
-**standalone method**: [`@octokit/request`](https://github.com/octokit/request.js#readme)
-
-An http request can be sent using `octokit.request`. It's API is matching GitHub's REST API documentation 1:1.
+You can call the GitHub REST API directly using `octokit.request`. The `reqeust` API matches GitHub's REST API documentation 1:1 so anything you see there, you can call using `request`. See [`@octokit/request`](https://github.com/octokit/request.js#readme) for all the details.
 
 Example: [Create an issue](https://docs.github.com/en/rest/reference/issues#create-an-issue)
 
 [![Screenshot of REST API reference documentation for Create an issue](assets/create-an-issue-reference.png)](https://docs.github.com/en/rest/reference/issues#create-an-issue)
 
-The according API to send the request with an `octokit` instance:
+The `octokit.request` API call corresponding to that issue creation documentation looks like this:
 
 ```js
```

Comment:
I like adding the link to the documentation, I think it's a good best practice. What do you think about adding that to the example?

````suggestion
```js
// https://docs.github.com/en/rest/reference/issues#create-an-issue
````